services/Cargo.toml

```diff
@@ -29,6 +29,7 @@ mpart-async = "0.5"
 num-traits = "0.2"
 paste = "1.0"
 postgres-types = { version = "0.2", features = ["derive"], optional = true }
+proj = "0.22"
```

Comment:
you imported proj but never used it.

Reply:
5b7e4b5da45b96e3cdb67d8226bf25f4e01aca9c